src/pages/playground/Playground.tsx

```diff
@@ -466,7 +466,8 @@ const Playground: React.FC<PlaygroundProps> = ({ workspaceLocation = 'playground
     );
   }, [dispatch, persistenceUser, persistenceFile, persistenceIsDirty]);
 
-  const githubOctokitObject = useSelector((store: any) => store.session.githubOctokitObject);
+  // FIXME: Remove any
```

Comment:
is it possible to type this octokit object properly?

Reply:
I just looked into it, seems like the selector is actually properly typed. It's the component that consumes the octokit object that is improperly typed. I've fixed this in e6fcdbd43b6199ea4472506b898876bfc2ce746e.